narwhals/_pandas_like/series.py

```diff
@@ -654,6 +653,13 @@ def is_null(self) -> PandasLikeSeries:
         ser = self._native_series
         return self._from_native_series(ser.isna())
 
+    def is_nan(self) -> PandasLikeSeries:
+        ser = self._native_series
+        if self.dtype.is_numeric():
+            return self._from_native_series(ser != ser)  # noqa: PLR0124
+        msg = f"`is_nan` is not supported for dtype {self.dtype}"
```

Comment:
Should this error message be extended to include a suggestion? 

"`is_nan` is not supported for dtype {self.dtype}, did you mean to use `is_null`?

Reply:
lots of love for kind error messages 💚